src/js/components/Menu/Menu.js

```diff
@@ -70,55 +70,45 @@ const Menu = props => {
   const iconColor = normalizeColor('control', theme);
   const align = dropProps.align || dropAlign;
   const buttonRefs = {};
+  const NONE = 'none';
```

Comment:
```suggestion
  const constants = {none: 'none'};
```
I'll be doing more refactoring around this area of constant strings, but for now, I think we can use this structure, so in this case, the constant will be approached as `constants.none`. If you have more constants, put them on there as well.